armi/physics/neutronics/parameters.py

```diff
@@ -118,35 +118,7 @@ def mgFlux(self, value):
         location=ParamLocation.AVERAGE,
         categories=[parameters.Category.detailedAxialExpansion],
     ) as pb:
-        # Neutronics reaction rate params
-        pb.defParam(
-            "rateAbs",
-            units="1/cm^3/s",
-            description="Total absorption rate in this block (fisson + capture).",
-        )
-
-        pb.defParam(
-            "rateCap",
-            units="1/cm^3/s",
-            description="Parasitic capture rate in this block.",
-        )
-
-        pb.defParam(
-            "rateFis", units="1/cm^3/s", description="Fission rate in this block."
-        )
-
-        pb.defParam(
-            "rateProdFis",
-            units="1/cm^3/s",
-            description="Production rate of neutrons from fission reactions (nu * fission source / k-eff)",
-        )
-
-        pb.defParam(
-            "rateProdN2n",
-            units="1/cm^3/s",
-            description="Production rate of neutrons from n2n reactions.",
-        )
-
+        # Neutronics reaction rate params that are not re-derived in mesh conversion
```

Comment:
Added some comments to explain what the category does. I removed some of these from that category so that we wouldn't waste time converting them since they're just going to be recalculated after the mapping is done.